packages/manager/modules/pci/src/projects/project/sidebar/project-list/project-list.html

```diff
@@ -19,8 +19,16 @@
         </div>
         <div class="mt-3" data-ng-if="$ctrl.projects && $ctrl.projects.length > 0">
             <strong class="text-white" data-translate="cloud_sidebar_projectlist_title"></strong>
+            <div class="oui-search mt-2">
+                <div class="oui-input-group oui-input-group_search">
+                    <input type="text" class="oui-input pr-4" data-ng-model="$ctrl.searchText" placeholder="{{:: 'cloud_sidebar_projectlist_title_search' | translate }}">
+                    <button class="oui-button" type="button">
+                        <span class="oui-icon oui-icon-search_extra-thin" aria-hidden="true"></span>
```

Comment:
`<span class="sr-only">This will launch the search, yo !</span>`

Reply:
`<button>` is used in an aesthetic purpose only which is not a perfect solution in my opinion.

My suggestion is reflecting the CX specification directly on the `<oui-search>` component (if possible).

It could be addressed in a separate pull request in order to process to the merge.